Source/SwiftLintBuiltInRules/Rules/Idiomatic/FinalClassRule.swift

```diff
@@ -0,0 +1,64 @@
+import SwiftSyntax
+
+@SwiftSyntaxRule
+struct FinalClassRule: Rule {
+    var configuration = SeverityConfiguration<Self>(.warning)
+
+    static let description = RuleDescription(
+        identifier: "final_class",
+        name: "Final Class",
+        description: "Prefer `static` over `final class`",
+        kind: .idiomatic,
+        nonTriggeringExamples: [
+            Example("""
+            class C {
+                static func f() {}
+            }
+            """),
+            Example("""
+            class C {
+                class func f() {}
+            }
+            """),
+            Example("""
+            final class C {}
+            """),
+        ],
+        triggeringExamples: [
+            Example("""
+            class C {
+                final class func f() {}
```

Comment:
Please add violation markers `↓`:
```suggestion
                ↓final class func f() {}
```